app/services/markdown_to_html.rb

```diff
@@ -0,0 +1,8 @@
+class MarkdownToHtml
```

Comment:
theres no rspec tests to cover this class, it needs to be well tested as its an important part of the site